### PR TITLE
Added tests for `overridemanager` package

### PR DESCRIPTION
**Pull request by 1Shubham7**
**What type of PR is this?**

/kind test

**What this PR does / why we need it**:

I have added unit tests for the `cloud/pkg/controllermanager/edgeapplication/overridemanager` package.

**Which issue(s) this PR fixes**:

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Comment by 1Shubham7** [comment]
locally the tests run fine and there are no linting issues :(

**Review by wbc6080 — COMMENTED** [review]
Have you rebased the latest code? We have completed the API migration in #5760 . The original `pkg/api` has been migrated to the `staging` directory.
xq1

File: cloud/pkg/controllermanager/edgeapplication/overridemanager/envoverrider_test.go [hunk 1]
```diff
@@ -0,0 +1,393 @@
...
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package overridemanager
+
+import (
+	"testing"
+
+	"github.com/kubeedge/kubeedge/pkg/apis/apps/v1alpha1"
```
Comment (wbc6080) [inline cloud/pkg/controllermanager/edgeapplication/overridemanager/envoverrider_test.go:22]:
```suggestion
	"github.com/kubeedge/api/apis/apps/v1alpha1"
```

File: cloud/pkg/controllermanager/edgeapplication/overridemanager/overridemanager_test.go [hunk 2]
```diff
@@ -0,0 +1,253 @@
...
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package overridemanager
+
+import (
+	"testing"
+
+	appsv1alpha1 "github.com/kubeedge/kubeedge/pkg/apis/apps/v1alpha1"
```
Comment (wbc6080) [inline cloud/pkg/controllermanager/edgeapplication/overridemanager/overridemanager_test.go:22]:
```suggestion
	appsv1alpha1 "github.com/kubeedge/api/apis/apps/v1alpha1"
```

File: cloud/pkg/controllermanager/edgeapplication/overridemanager/replicasoverrider_test.go [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package overridemanager
+
+import (
+	"testing"
+
+	apppsv1alpha1 "github.com/kubeedge/kubeedge/pkg/apis/apps/v1alpha1"
```
Comment (wbc6080) [inline cloud/pkg/controllermanager/edgeapplication/overridemanager/replicasoverrider_test.go:22]:
```suggestion
	apppsv1alpha1 "github.com/kubeedge/api/apis/apps/v1alpha1"
```

File: cloud/pkg/controllermanager/edgeapplication/overridemanager/resourcesoverrider_test.go [hunk 4]
```diff
@@ -0,0 +1,549 @@
...
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package overridemanager
+
+import (
+	"testing"
+
+	"github.com/kubeedge/kubeedge/pkg/apis/apps/v1alpha1"
```
Comment (wbc6080) [inline cloud/pkg/controllermanager/edgeapplication/overridemanager/resourcesoverrider_test.go:22]:
```suggestion
	"github.com/kubeedge/api/apis/apps/v1alpha1"
```

File: cloud/pkg/controllermanager/edgeapplication/overridemanager/envoverrider_test.go [hunk 5]
```diff
@@ -0,0 +1,393 @@
...
+
+package overridemanager
+
+import (
+	"testing"
+
+	"github.com/kubeedge/api/apis/apps/v1alpha1"
+	"github.com/stretchr/testify/assert"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
```
Comment (wbc6080) [inline cloud/pkg/controllermanager/edgeapplication/overridemanager/envoverrider_test.go:25]:
```suggestion
	"testing"

	"github.com/stretchr/testify/assert"
	corev1 "k8s.io/api/core/v1"
	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
	
	"github.com/kubeedge/api/apis/apps/v1alpha1"
```

Need to sort the import. Other files also need to be modified like this. @1Shubham7

Comment (1Shubham7) [reply cloud/pkg/controllermanager/edgeapplication/overridemanager/envoverrider_test.go]: yup, got it. sorry I didn't check that

**Review by wbc6080 — APPROVED** [review]
/lgtm

**Review by Shelley-BaoYue — COMMENTED** [review]
/lgtm

**Comment by 1Shubham7** [comment]
@WillardHu please review this one as well :)

**Pull request merged**